keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/constraints/Ec2CanaryConstraintDeployHandler.kt

```diff
@@ -16,7 +16,7 @@ import com.netflix.spinnaker.keel.constraints.toStageBase
 import com.netflix.spinnaker.keel.ec2.resolvers.ImageResolver
 import com.netflix.spinnaker.keel.events.Task
 import com.netflix.spinnaker.keel.model.parseMoniker
-import com.netflix.spinnaker.keel.model.toEchoNotification
+import com.netflix.spinnaker.keel.model.toOrcaNotification
```

Comment:
I renamed this because it is actually an orca-specific format to represent notifications, and not what the echo API uses (orca does the translation).